### PR TITLE
Add ability to deploy CO on Hosted Cluster through OLM

### PR DESCRIPTION
**Pull request by Vincent056**
Added Subscription file needed for deploying CO on HyperShift Hosted Cluster through OLM, it can be used from both upstream or downstream sources.

### CODE REVIEW

**Comment by Vincent056** [comment]
This needs https://github.com/ComplianceAsCode/compliance-operator/pull/331 to merge to work

**Review by jhrozek — APPROVED** [review]
/lgtm
looks good, but should be documented carefully downstream

File: doc/usage.md [hunk 1]
```diff
@@ -616,4 +616,39 @@ settingsRef:
...
- ```
\ No newline at end of file
+ ```
+
+## How to Use Compliance Operator with HyperShift Hosted Cluster
+
+Compliance Operator is able to run platform scan on the HyperShift Hosted cluster
+without any tailoredProfile. Any unsupport rules will be hidden from the `ComplianceCheckResult`.
+
+However, you need to use a sepcial subscription file to install Compliance Operator on the
```
Comment (GroceryBoyJr) [inline doc/usage.md:626]:
```suggestion
However, you need to use a special subscription file to install Compliance Operator on the
```

File: doc/usage.md [hunk 2]
```diff
@@ -616,4 +616,39 @@ settingsRef:
  name: default
  kind: ScanSetting
  apiGroup: compliance.openshift.io/v1alpha1
- ```
\ No newline at end of file
+ ```
+
+## How to Use Compliance Operator with HyperShift Hosted Cluster
+
+Compliance Operator is able to run platform scan on the HyperShift Hosted cluster
```
Comment (GroceryBoyJr) [inline doc/usage.md:623]:
```suggestion
Compliance Operator is able to run a platform scan on the HyperShift Hosted cluster
```

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM, only two small nits. Will discuss with @sheriff-rh regarding downstream doc options.

**Review by jhrozek — APPROVED** [review]
/lgtm

**Review by jhrozek — APPROVED** [review]
/lgtm
only PX ack missing now!

**Pull request merged**